test/Microsoft.Data.Sqlite.Tests/TestUtilities/WinSqlite3TestFixture.cs

```diff
@@ -1,27 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Reflection;
-using Microsoft.Data.Sqlite.Internal;
-using Xunit.Abstractions;
-using Xunit.Sdk;
-
-[assembly: Xunit.TestFramework("Microsoft.Data.Sqlite.WinSqlite3TestFramework", "Microsoft.Data.Sqlite.Tests")]
-
-namespace Microsoft.Data.Sqlite
-{
-    public class WinSqlite3TestFramework : XunitTestFramework
```

Comment:
winsqlite3.dll is supported, but it's done differently.  I didn't set that up as part of the proof-of-concept stuff I was doing, but it's not difficult.